src/v0/form/js/main.js

```diff
@@ -448,6 +448,36 @@ $('.resolution-options').hide();
       $(this).closest('label').toggleClass('active');
   });
 
+/**
+ * Shows and hides mortgage subissue followup questions based on answers to
+ * preceeding trigger questions
+ */
+$( '.mortgage-servicer-subissue-question__radio' ).on( 'change', '.radio_input', function() {
+  var mortageQuestionModel = {},
+      thisQuestionID = $( this ).parents( '.mortgage-servicer-subissue-question__radio' ).attr('id'),
+      $folowupQuestions = $( '[data-followup-to*="' + thisQuestionID + '"]' );
+  $( '.mortgage-servicer-subissue-question__radio' ).each( function() {
+    var questionID = $( this ).attr( 'id' ),
+        questionValue = $(this).find( '.radio_input:checked' ).val();
+    mortageQuestionModel[ questionID ] = questionValue;
+  });
+  $folowupQuestions.each( function() {
+    var $question = $( this ),
+        triggerQuestionIDs = $question.attr( 'data-followup-to' ).split( ' ' );
+    // If ANY of the follow-up question's triggers are answered "yes", show the
+    // follow-up. Otherwise, hide the follow-up.
+    for (var i = 0; i < triggerQuestionIDs.length; i++) {
+      var triggerAnswer = mortageQuestionModel[ triggerQuestionIDs[ i ] ];
+      if ( triggerAnswer === 'yes' ) {
+        $question.slideDown();
+        break;
+      } else {
```

Comment:
`else` is unnecessary because of the `break` in the if statement. (Note: the linter may complain about the break statement, but we can refactor later.)